### PR TITLE
Added a db config option to change the table prefix

### PR DESCRIPTION
**Pull request by adamsong**
Having to find/replace all db queries to get them to work with the schemas provided in the main repo is dumb.

### CODE REVIEW

File: models/Discord/Commands/DiscordCommandWhoIs.js [hunk 1]
```diff
@@ -33,9 +33,9 @@ class DiscordCommandWhoIs extends DiscordCommand {
       var userID = auser
       
       if(userID) {
-        getByDiscordID(connection, userID, message)
+        getByDiscordID(connection, userID, message, dbSubsystem.format_table_name('player'))
       } else {
-        getByCkey(connection, provided_ckey, message)
+        getByCkey(connection, provided_ckey, message, dbSubsystem.format_table_name('player'))
```
Comment (alexkar598) [inline models/Discord/Commands/DiscordCommandWhoIs.js:38]: WTF is this, set it in the query

**Review by alexkar598 — CHANGES_REQUESTED** [review]

File: models/Discord/Commands/DiscordCommandWhoIs.js (hunk 1 above)
Comment (adamsong) [reply models/Discord/Commands/DiscordCommandWhoIs.js:38]: The function doesn't have access to dbSubsystem

Comment (alexkar598) [reply models/Discord/Commands/DiscordCommandWhoIs.js:38]: Then make it have it. Move the function in scope.

**Comment by alexkar598** [comment]
Stale.

**Review by alexkar598 — CHANGES_REQUESTED** [review]
Missing queries in verification endpoint

**Review by alexkar598 — APPROVED** [review]

**Pull request merged**